TESTING.md

```diff
@@ -34,15 +34,14 @@ OpenSearch uses [jUnit](https://junit.org/junit5/) for testing, it also uses ran
 - [Test coverage analysis](#test-coverage-analysis)
 - [Building with extra plugins](#building-with-extra-plugins)
 - [Environment misc](#environment-misc)
-- [Benchmarking](#benchmarking)
 
 # Requirements
 
 You will need the following pieces of software to run these tests:
 
 - Docker & Docker Compose
 - Vagrant
-- JDK 14
+- JDK 11
```

Comment:
Are we using 11 for the GA artifacts build?

Reply:
No, not for GA. This is on main and see https://github.com/opensearch-project/OpenSearch/pull/636.